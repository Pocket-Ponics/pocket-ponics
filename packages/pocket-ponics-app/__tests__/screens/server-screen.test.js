import React from 'react'
import { shallow } from 'enzyme'

import ServerScreen from '../../screens/server-screen'

describe('Server Screen', () => {
	test('renders correctly', () => {
		const tree = shallow(
			<ServerScreen />
		).html()
		expect(tree).toMatchSnapshot()
	})
})