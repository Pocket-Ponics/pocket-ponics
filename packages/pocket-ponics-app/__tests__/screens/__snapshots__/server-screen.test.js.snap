// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Server Screen renders correctly 1`] = `"<View style=\\"0:[object Object];1:[object Object]\\"><View style=\\"flex:1;align-items:center;justify-content:center;padding:20px;margin-top:40px;width:100%\\"><Text style=\\"color:#FFFFFF;font-size:32px;margin-bottom:10px;margin-top:10px;text-align:center\\">Select a Custom Server</Text><Text style=\\"color:#FFFFFF;font-size:18px;margin-bottom:20px\\">For advanced users who have a personal Pocket Ponics server. Most users should just use the default server</Text><RCTScrollView data=\\"[object Object]\\" style=\\"margin:0;width:100%\\" extraData=\\"[object Object]\\" initialNumToRender=\\"10\\" maxToRenderPerBatch=\\"10\\" scrollEventThrottle=\\"50\\" updateCellsBatchingPeriod=\\"50\\" windowSize=\\"21\\" numColumns=\\"1\\" viewabilityConfigCallbackPairs=\\"\\" stickyHeaderIndices=\\"\\"><View><View><View style=\\"border-radius:5px;padding:10px;border-width:3px;border-color:#456E31;opacity:1\\"><Text style=\\"color:#FFFFFF;font-size:22px\\">Default Server</Text><Text style=\\"color:#FFFFFF;font-size:22px\\">Host: Default host</Text><Text style=\\"color:#FFFFFF;font-size:22px\\">Port: Default port</Text></View></View><View><View><Text style=\\"color:#FFFFFF;font-size:32px;margin-bottom:10px;margin-top:10px;text-align:center\\">New Custom Server</Text><TextInput underlineColorAndroid=\\"transparent\\" style=\\"width:100%;height:45px;border-radius:45px;font-size:16px;padding-left:45px;background-color:rgba(0, 0, 0, 0.35);color:#FFFFFF\\" placeholder=\\"Custom Server Name\\" placeholderTextColor=\\"#FFFFFF\\" value=\\"\\"></TextInput><TextInput underlineColorAndroid=\\"transparent\\" style=\\"width:100%;height:45px;border-radius:45px;font-size:16px;padding-left:45px;background-color:rgba(0, 0, 0, 0.35);color:#FFFFFF\\" placeholder=\\"Custom Server Host\\" placeholderTextColor=\\"#FFFFFF\\" value=\\"\\"></TextInput><TextInput underlineColorAndroid=\\"transparent\\" style=\\"width:100%;height:45px;border-radius:45px;font-size:16px;padding-left:45px;background-color:rgba(0, 0, 0, 0.35);color:#FFFFFF\\" placeholder=\\"Custom Server Port\\" placeholderTextColor=\\"#FFFFFF\\" value=\\"\\"></TextInput><View style=\\"align-items:center;opacity:1\\"><Text style=\\"color:#84AB71;font-size:22px\\">Add Server</Text></View></View></View></View></RCTScrollView><View style=\\"background-color:#456E31;border-radius:5px;padding:10px;margin:10px;width:100%;align-items:center;opacity:1\\"><Text style=\\"color:#FFFFFF;font-size:22px\\">Save Server Info</Text></View></View></View>"`;
